graphql.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module '*.graphql' {
```

Comment:
I encountered a TS error after the initial conversion of `alert-banner`:

<img width="989" alt="CleanShot 2021-08-16 at 12 12 59@2x" src="https://user-images.githubusercontent.com/4624598/129603349-c3682725-33ba-4f61-9cf8-bf7e73346055.png">

Based on [this issue](https://github.com/apollographql/graphql-tag/issues/59), I _think_ this is the right solution for our use case. Wasn't sure whether it belonged somewhere else... Maybe https://github.com/hashicorp/nextjs-scripts/blob/main/packages/types/index.d.ts ?

Also noticed that https://github.com/hashicorp/hashicorp-www-next/blob/main/types/global.d.ts has something similar (possibly duplicative of `platform-types`?)

Reply:
Yeah, I think this would be a good addition to our types package! https://github.com/hashicorp/nextjs-scripts/blob/main/packages/types/index.d.ts

Reply:
👍 - have opened https://github.com/hashicorp/nextjs-scripts/pull/198